src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModel.java

```diff
@@ -193,18 +193,18 @@ public ReferenceConfidenceModel(final SampleList samples,
         final String sampleName = readLikelihoods.getSample(0);
 
         final int globalRefOffset = refSpan.getStart() - activeRegion.getExtendedSpan().getStart();
-        for ( final ReadPileup pileup : refPileups ) {
+        for (int i = 0, size = refPileups.size(); i < size; i++) {
```

Comment:
Declare `refPileupsSize` as a final var before the loop instead of using multiple initialization within the loop.